draft-ietf-quic-transport.md

```diff
@@ -1943,6 +1943,11 @@ the client during connection establishment with a Retry packet (see
 {{validate-retry}}) or in a previous connection using the NEW_TOKEN frame (see
 {{validate-future}}).
 
+Until the client address is validated, servers SHOULD ensure that UDP datagrams
+containing Initial packets have UDP payloads of at least 1200 bytes.  Doing so
```

Comment:
I would revert this to what you had earlier... I don't think there's a good reason to not recommend padding all packets, since client behavior is uncertain here. The SHOULD here means that implementations can deviate from this recommendation, as long as they know what they're doing.

Reply:
Retry can validate the client's address, so the ping-pong could still happen in that case.

Reply:
@marten-seemann Wow. That's true. Addressed in bd089c9.